app/controllers/api/v1/invoices/item_invoice_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::Invoices::ItemInvoiceController < ApplicationController
```

Comment:
@tmikeschu  It is names Item invoice because it is calling .items on a particular invoice. An invoice has_many :items, through: invoice_items and an item has_many :invoices